code_challenges.rb

```diff
@@ -1,48 +1,81 @@
 # ASSESSMENT 4: Ruby Coding Practical Questions
-
 # --------------------1) Create a method that takes in an array and returns an array with only odd numbers sorted from least to greatest.
-
 fullArr1 = [4, 9, 0, '7', 8, true, 'hey', 7, 199, -9, false, 'hola']
 # Expected output: [-9, 7, 9, 199]
 fullArr2 = ['hello', 7, 23, -823, false, 78, nil, '67', 6, 'Number']
 # Expected output: [-823, 7, 23]
-
-
-
-
-
-# --------------------2) Create a class called Bike that is initialized with a model, wheels, and a frame size. The default number of wheels is 2. Create a get_info method that returns a sentence with all the data from the bike object.
-
-
-
-
-
+# def odd_nums array do |number, index|
+#   if (index %2 ==0) then
+#     fullArr1.push(number)
+#   end
+# end
```

Comment:
I'm not seeing a call for this function here. When I tried to create a call for it, we're getting a bunch of syntax errors.